server/routes/jwt-refresh.js

```diff
@@ -31,6 +31,8 @@ router.post('/', (req, res) => {
               }
             }
         );
+    } else {
```

Comment:
Up on line 9 we should be looking for the refresh token, not the access token. If the access token cooke has expired it won't be present, but the refresh token cookie could still be there and usable.

Reply:
Good point.  Currently, both the access_token and refresh_token are session based cookies.  But the access_token should expire when the jwt expires.  We should do this in a separate PR